test/e2e/workflow/deployment_test.go

```diff
@@ -438,6 +438,57 @@ var _ = Describe("NetworkAddonsConfig", func() {
 					}, 5*time.Minute, time.Second).Should(BeNil(), "Timed out waiting for nmstate-operator daemonset")
 				})
 
+				Context("when NAC is updated", func() {
+					BeforeEach(func() {
+						configSpec = cnao.NetworkAddonsConfigSpec{
+							NMState: &cnao.NMState{},
+							SelfSignConfiguration: &cnao.SelfSignConfiguration{
+								CARotateInterval:    "42h0m0s",
+								CAOverlapInterval:   "42h0m0s",
+								CertRotateInterval:  "42h0m0s",
+								CertOverlapInterval: "42h0m0s",
+							},
+						}
+					})
+					It("shouldn't update NMState CR", func() {
+						By("checking for NMState in nmstate namespace")
+						Eventually(func() error {
```

Comment:
hmm, how is it different from `CheckNMStateOperatorIsReady` which you use just after this `Eventually`?
In any case I would move to a function (like `CheckNMStateOperatorIsReady`) to increase documentation..

Reply:
You're right, the Eventually should not be needed

Reply:
[Done](https://github.com/kubevirt/cluster-network-addons-operator/compare/08d160a48ed4b089cc654761fd1112b3bb3a3e68..35f776d9be08fce4039c5feeb2a2cfa0f8faec9e)